src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -2102,21 +2105,6 @@ private void recordMetrics(SliceQueryFilter filter) {
         metric.liveReadHistogram.update(filter.lastReadLive());
         metric.tombstonesReadHistogram.update(filter.lastReadTombstones());
```

Comment:
right now, the droppableTombstones and ranged tombstone metrics will be updated in-situ, but these metrics will still be updated after the fact, right? any reason you don't move these into `CountingCellIterator ` and `ColumnCounter`?